go.mod

```diff
@@ -23,23 +22,28 @@ require (
 	github.com/miekg/dns v1.1.30
 	github.com/pborman/getopt v0.0.0-20190409184431-ee0cd42419d3
 	github.com/peterbourgon/ff/v2 v2.0.0
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/stretchr/testify v1.5.1 // indirect
 	github.com/tailscale/depaware v0.0.0-20201214215404-77d1e9757027
 	github.com/tailscale/wireguard-go v0.0.0-20210210202228-3cc76ed5f222
 	github.com/tcnksm/go-httpstat v0.2.0
 	github.com/toqueteos/webbrowser v1.2.0
 	go4.org/mem v0.0.0-20201119185036-c04c5a6ff174
 	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad
 	golang.org/x/net v0.0.0-20210220033124-5f55cee0dc0d
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
```

Comment:
Why these various changes that aren't netstack?

Reply:
`go mod tidy`, which was necessary to clean up the mess from before.

we don't have CI enforcing `go mod tidy`-ness and people forget. (it's also just not too common).

I just pushed a `go mod tidy` before this change to minimize this one. Will rebase.

Reply:
Ah, because this chaos:

```
module golang.org/x/oauth2

go 1.11

require (
        cloud.google.com/go v0.65.0
        golang.org/x/net v0.0.0-20200822124328-c89045814202
        google.golang.org/appengine v1.6.6
)
```

And we ourselves use `oauth2`, so when it bumps we get lots of new google deps.

Reply:
I just downgraded some things to match what they were at HEAD^ but still not quite sure why some remaining changes exist.

I'll figure it out before submitting, if I can't get this down to only obvious changes.

Reply:
oh! the new additions are `// indirect' lines. See https://github.com/golang/go/wiki/Modules#why-does-go-mod-tidy-record-indirect-and-test-dependencies-in-my-gomod

When we removed gvisor, many things (that we already depended on indirectly) went from being specified via gvisor to no longer being specified, so the indirect lines specify them.

Will rebase this upon some separate golang.org/x/* updates so this diff is more obvious.